src/components/Captures/CaptureTable.js

```diff
@@ -258,26 +260,35 @@ const CaptureTable = () => {
           </TableRow>
         </TableHead>
         <TableBody data-testid="captures-table-body">
-          {captures &&
-            captures.map((capture) => (
-              <TableRow
-                key={capture.id}
-                onClick={createToggleDrawerHandler(capture.id)}
-                className={classes.tableRow}
-              >
-                {columns.map(({ attr, renderer }) => (
-                  <TableCell key={attr}>
-                    {formatCell(
-                      capture,
-                      speciesLookup,
-                      captureTagLookup[capture.id] || [],
-                      attr,
-                      renderer
-                    )}
-                  </TableCell>
-                ))}
-              </TableRow>
-            ))}
+          {captures.length === 0 ? (
+            <div
+              style={{ position: 'relative', left: '300%', bottom: '-179px' }}
```

Comment:
Move the style details to CaptureTable.styles.js as a new class that is referenced with an attribute like `className={classes.loadingIndicator}`.